janusgraph-test/src/test/java/org/janusgraph/core/ConfiguredGraphFactoryTest.java

```diff
@@ -278,5 +286,54 @@ public void ensureCallingGraphCloseResultsInNewGraphReferenceOnNextCallToOpen()
             ConfiguredGraphFactory.close("graph1");
         }
     }
+
+    @Test
+    public void dropShouldCleanUpTraversalSourceAndBindings() throws Exception {
```

Comment:
This test seems to be flaky. I encountered "ConfiguredGraphFactoryTest.dropShouldCleanUpTraversalSourceAndBindings:312 expected: <true> but was: <false>" twice and had to retry to make it pass.